src/content/docs/specs/episodes/get-all.mdx

```diff
@@ -0,0 +1,262 @@
+---
+title: Get all episodes
+description: Get all episodes for a user
+sidebar:
+  order: 3
+  badge:
+    text: Core
+    variant: caution
+---
+
+import CoreAction from "@partials/_core-action.mdx";
+
+<CoreAction />
+
+```http title="Endpoint"
+GET /v1/episodes
+```
+
+This endpoint enables clients to return all episode information relating to the authenticated user. It returns pagination information and an array of `episodes`.
+
+While supported, this endpoint is expected to be used rarely in practice. More often, episodes are retrieved per queue (TBD) or per subscription (TBD).
+
+## Response fields
+
+### Metadata
+
+| Field      | Type   | Required? | Description                                      |
+| ---------- | ------ | --------- | ------------------------------------------------ |
+| `total`    | Number | Yes       | The total number of objects returned by the call |
+| `page`     | Number | Yes       | The number of the page returned in the call      |
+| `per_page` | Number | Yes       | The number of results returned per page          |
+| `next`     | String | No        | The URL for the next page of results             |
+| `previous` | String | No        | The URL for the previous page of results         |
+
+### Episode fields
+
+| Group      | Field                               | Type            | Required? | Description                                                                                                                                                                                     |
```

Comment:
Having this table seems a bit superfluous actually with the 'important fields' table on the Overview page. Do you think it would be safe to drop it here, and add a reference to the other page, instead @Sporiff?